polaris-tokens/src/tokens.ts

```diff
@@ -10,6 +10,8 @@ import {typography} from './token-groups/typography';
 import {zIndex} from './token-groups/zIndex';
 import {tokensToRems} from './utilities';
 
+export type {SpacingTokens} from './token-groups/spacing';
```

Comment:
```suggestion
export type {SpacingTokenName} from './token-groups/spacing';
```

Per the above suggestion